src/Searchbar.vue

```diff
@@ -88,12 +89,17 @@ export default {
       methods: {
         highlight(value, phrase) {
           function getMatchIntervals() {
-            const keywords = phrase.split(' ').filter(keyword => keyword !== '');
+            const keywords = phrase.split(' ')
+              .filter(keyword => keyword !== '')
+              .map(searchKeyword => searchKeyword.replace(/[.*+?^${}()|[\]\\]/g, '\\$&'));
```

Comment:
Shall we `map` to get `regexes` here?
```js
const regexes = phrase.split(' ')
  .filter(searchKeyword => searchKeyword !== '')
  .map(searchKeyword => searchKeyword.replace(/[.*+?^${}()|[\]\\]/g, '\\$&'))
  .map(searchKeyword => new RegExp(`(${searchKeyword})`, 'gi'));
```

Then we also disambiguate `keywords` from `item.keywords`.